README.md

```diff
@@ -176,14 +176,25 @@ import puppeteer from 'puppeteer';
 
 **1. Uses Headless mode**
 
-Puppeteer launches Chromium in
+By default Puppeteer launches Chromium in
 [headless mode](https://developers.google.com/web/updates/2017/04/headless-chrome).
+
+> Note: Deprecated please consider using `headless: 'new'`
+> See example bellow.
```

Comment:
IMHO this seems to suggest that “uses Headless mode” (section title) is somehow deprecated

Reply:
How does this sound/look?
![image](https://user-images.githubusercontent.com/34244704/233051163-f785acf5-8153-46b5-bf09-60c050837248.png)

Reply:
LGTM (reviewed offline)